src/components/dialogs/DeduplicationSummaryDialog.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { injectIntl } from 'react-intl';
+import Button from '@material-ui/core/Button';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import { formatMessage } from '@openimis/fe-core';
+import { withTheme, withStyles } from '@material-ui/core/styles';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import DeduplicationSummaryTable from '../tables/DeduplicationSummaryTable';
+import { fetchDeduplicationSummary } from '../../actions';
+
+const styles = (theme) => ({
+  item: theme.paper.item,
+});
+
+function DeduplicationSummaryDialog({
+  intl,
+  benefitPlan,
+  handleClose,
+  showSummaryDialog,
+  selectedValues,
+}) {
+  if (!benefitPlan) return null;
+
+  const columns = selectedValues.map((value) => value.id);
+  const columnParam = `columns: ${JSON.stringify(columns)}`;
+
+  return (
+    <Dialog
+      open={showSummaryDialog}
+      onClose={handleClose}
+      PaperProps={{
+        style: {
+          width: 900,
+          maxWidth: 900,
+        },
+      }}
+    >
+      <DialogTitle
+        style={{
+          marginTop: '10px',
+        }}
+      >
+        {formatMessage(intl, 'deduplication', 'deduplicate.summary.title')}
+      </DialogTitle>
+      <DialogContent>
+        <DeduplicationSummaryTable
+          columnParam={columnParam}
+          benefitPlan={benefitPlan}
+          fetchDeduplicationSummary={fetchDeduplicationSummary}
+        />
+      </DialogContent>
+      <DialogActions
+        style={{
+          display: 'inline',
+          paddingLeft: '10px',
+          marginTop: '25px',
+          marginBottom: '15px',
```

Comment:
What do you think on moving css to utils/styles? It would improve the readability little bit.